chart/templates/configmap.yaml

```diff
@@ -11,3 +11,19 @@ data:
   default_resource_requests_memory: "128Mi"
   always_send_finished_event: "false"
   enable_kubernetes_api_access: "{{ .Values.jobConfig.enableKubernetesApiAccess | default false }}"
+  keptn_api_endpoint: {{ (printf "%s://%s/api" .Values.remoteControlPlane.api.protocol .Values.remoteControlPlane.api.hostname) }}
+  init_container_configuration_endpoint:
+    {{- if .Values.remoteControlPlane.enabled }}
+      {{ (printf "%s://%s/api/configuration-service" .Values.remoteControlPlane.api.protocol .Values.remoteControlPlane.api.hostname) }}
+    {{- else }}
+      "http://configuration-service:8080"
+    {{- end }}
```

Comment:
Let's also remove the if else here. REmote control plane should be the default, end we no longer access configurations-ervice directly.